app/Http/Controllers/RoleController.php

```diff
@@ -86,12 +87,22 @@ public function update(RoleRequest $request, Role $role): RoleResource
      * Remove the specified resource from storage.
      *
      * @param Role $role
-     * @return \Illuminate\Http\JsonResponse
+     * @return JsonResponse
      */
-    public function destroy(Role $role): \Illuminate\Http\JsonResponse
+    public function destroy(Role $role): JsonResponse
     {
         $role->delete();
 
         return response()->json(["Role deleted"]);
     }
+
+    /**
+     * Get current user roles
+     *
+     * @return JsonResponse
+     */
+    public function getRoles(): JsonResponse
+    {
+        return response()->json(auth()->user()->role);
+    }
```

Comment:
Should it use the auth('sanctum') guard?
Additionally should it have a auth('sanctum)->check() to check if user is logged in

Reply:
I see no reason to not